### DISCUSSION TITLE
Instructions on how to "include note in another note" are missing.

### DISCUSSION THREAD

**Discussion by LeoStaley** [TriliumNext/Trilium#1905] [Q&A]
In the Trilium demo, there's a note that shows that it is possible to include one note inside another note, but there are no instructions on how to do it. 

I eventually found an issue here on GitHub mentioning that there is now a way to add a keyboard shortcut for it, but no information in the documentation, either here, or in the demo, on how to do it normally, without the keyboard shortcut.

Indeed, I can't figure out how to do ANY of the formatting examples shown. Could you include in the demo instructions on how to actually insert the formatting features mentioned? What I expect is to right click and have a context menu show up for what to insert or change, but all I get with right clicking is copy/paste options.

**Comment by zadam** [comment]
Hi, so there are 2 kind of toolbars, one is opened when selecting text:

![image](https://user-images.githubusercontent.com/617641/116597579-10c30600-a926-11eb-8500-60c3d48a2302.png)

another is for whole blocks and can be triggered by clicking on that small icon on the left side of the current line:

![image](https://user-images.githubusercontent.com/617641/116597728-42d46800-a926-11eb-90be-abf76e3b9138.png)

Button to include a note is circled in red.
